sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -245,7 +245,6 @@ protected AmqpConnectionScope()
         {
             Argument.AssertNotNullOrEmpty(consumerGroup, nameof(consumerGroup));
             Argument.AssertNotNullOrEmpty(partitionId, nameof(partitionId));
-            Argument.AssertNotNull(eventPosition, nameof(eventPosition));
```

Comment:
Am assuming `CreateReceivingLinkAsync` is able to handle null event position?

Reply:
Event Position is now a struct and, therefore, cannot be null.